game/src/main/scala/io/aigar/game/Vector2Utils.scala

```diff
@@ -1,9 +1,16 @@
 package io.aigar.game
 
+import scala.util.Random
+import scala.math.{Pi, cos, sin}
 import com.github.jpbetz.subspace._
 import io.aigar.game.serializable.Position
 
 object Vector2Utils {
+  def randomUnitVector: Vector2 = {
+    val angle = Random.nextFloat * 2f * Pi.toFloat
+    Vector2(cos(angle).toFloat, sin(angle).toFloat)
```

Comment:
Comes directly from:
 ![the trigonometric circle](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Unit_circle_angles_color.svg/300px-Unit_circle_angles_color.svg.png)
But we can talk about this offline if you want.